wrapper/src/main/java/software/amazon/jdbc/wrapper/ExceptionManagerService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.wrapper;
+
+import software.amazon.jdbc.exceptions.ExceptionHandler;
+
+public interface ExceptionManagerService {
+
+  boolean isLoginException(Throwable throwable);
+
+  boolean isNetworkException(Throwable throwable);
+
+  void setCustomExceptionHandler(ExceptionHandler handler);
```

Comment:
What part of the driver uses this method? How user app can setup a custom handler?